source/includes/_tipser-elements-analytics.md

```diff
@@ -74,6 +74,34 @@ The code above assumes that you use <a href="https://developers.google.com/analy
 
 For the instructions how to setup analytics.js script on your site and connect it to your GA account, refer to the <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs" target="_blank">official documentation</a>.
 
+## Typical use case: Facebook Pixel
+
+Facebook Pixel is also an option to tunnel Tipser Analytics events, here's a snippet:
```

Comment:
This sentence wasn't 100% clear for me the first time I read it.

Maybe we could just keep it as a mirror of GA section:
"In case you want to forward Tipser Analytics events to your Facebook Pixel account, you can use the following code snippet:"

(I deliberately replaced the word "tunnel" with "forward" as I believe it sounds more familiar and I think we should do the same in the GA section)